run_preprocessing.py

```diff
@@ -9,6 +9,14 @@
 from typing import Tuple
 
 
+def save_to_csv(
```

Comment:
I'm not entirely sure if we want to do this for the whole graph (will take Gbs)? could we make it optional (in DataConfig)?

Reply:
Yes, I did it for debugging purposes. I now made it optional.